index.d.ts

```diff
@@ -2,13 +2,14 @@ export = dashjs;
 export as namespace dashjs;
 
 declare namespace dashjs {
-    class Debug {
+    interface Debug {
```

Comment:
Technically `Debug` is an interface, not a class. This is because `Debug` objects are created with object literals (`{}`), not with a constructor (`new Debug()`).